### ISSUE TITLE
MAINT: EXOSCALE upgrade dependencies 

### ISSUE THREAD

**Issue by tlimoncelli** [StackExchange/dnscontrol#1564]
@pierre-emmanuelJ

It looks like this provider depends on a module that has a newer version.  Would you please test your provider with the newer version and submit a PR?

```
$ go list -mod=mod -u -m -json all | go-mod-outdated -update -direct
+---------------------------------+---------+-------------+--------+------------------+
|             MODULE              | VERSION | NEW VERSION | DIRECT | VALID TIMESTAMPS |
+---------------------------------+---------+-------------+--------+------------------+
| github.com/exoscale/egoscale    | v0.59.0 | v1.19.0     | true   | false            |
+---------------------------------+---------+-------------+--------+------------------+
```

Near the end of `docs/release-engineering.md` are 2 different techniques for finding dependencies that need to be upgraded.

TLDR: Please submit a PR to upgrade this dependency.  Basically you'll be running these commands:

```
$ git checkout master
$ git pull
$ git checkout -b INSERT_BRANCH_NAME
$ go get INSERT_MODULE_NAME_HERE
$ go mod tidy
$ cd integrationTest
$ go test -v -verbose -provider INSERT_PROVIDER_NAME_HERE
```

Please fix any issues then submit a PR.

Thanks!

**Comment by tlimoncelli** [comment]
@pierre-emmanuelJ Friendly ping?

**Comment by tlimoncelli** [comment]
@pierre-emmanuelJ Friendly ping?

**Comment by pierre-emmanuelJ** [comment]
Thanks for your time, 

v1.19.0 is a wrong version pushed accidentally, latest version is: [v0.102.2](https://github.com/exoscale/egoscale/releases/tag/v0.102.2)

I put the task in our internal backlog to update the egoscale library and re-run the integration test...

We have an ongoing version coming soon in alpha now I think it's better to do the effort with this one, once it's a bit more stable.
https://github.com/exoscale/egoscale/tree/master/v3

**Comment by tlimoncelli** [comment]
Thanks for the update!

**Comment by tlimoncelli** [comment]
@pierre-emmanuelJ Any update?
